frontend/Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /home/app
 
 COPY --chown=app:app ./package.json ./bun.lockb /home/app/
 
-RUN bun install
+RUN bun install --production --frozen-lockfile
```

Comment:
🤔 do we need the prod flag in localdev?

Reply:
maybe co-pilot agrees with me 🙃
<img width="469" alt="image" src="https://github.com/user-attachments/assets/0324025a-45b6-4dc1-a697-099218b580ce">

Reply:
this is what the dev tools look like with prod flag 👎 
<img width="1165" alt="image" src="https://github.com/user-attachments/assets/b66f3e22-73e4-450b-a7d6-d63ca6493755">

Reply:
this is what they should look like 👍 
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/18f7bfe7-845f-4b7d-87e9-b9d0c6cb5ffc">